packages/app-desktop/gui/ConfigScreen/configScreenUtilsImpl.tsx

```
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { AppType, SettingItem } from '@joplin/lib/models/Setting';
+
+const normalizedQuery = (searchQuery: string): string => {
+	return String(searchQuery || '').trim().toLowerCase();
+};
+
+const searchTerms = (searchQuery: string): string[] => {
+	const q = normalizedQuery(searchQuery);
```

ESLint

[error] Expected linebreaks to be 'LF' but found 'CRLF'.

(linebreak-style)